<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidget</class>
 <widget class="QWidget" name="SettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>561</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabSetPix">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>401</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-401</y>
        <width>542</width>
        <height>800</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>800</height>
       </size>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>521</width>
         <height>381</height>
        </rect>
       </property>
       <property name="title">
        <string>点设置</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>点的大小</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="linePixSize">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>481</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>点的背景颜色</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelColorPix">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>点的颜色</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbPixColorGet">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>130</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>获取颜色</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>521</width>
         <height>371</height>
        </rect>
       </property>
       <property name="title">
        <string>边设置</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>边的粗细</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdgeWidth">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>481</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>边的颜色</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelColorLine">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>53</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>边的颜色</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbEdgeColorGet">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>130</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>获取颜色</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSetdis">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>371</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-431</y>
        <width>542</width>
        <height>800</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>800</height>
       </size>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>521</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>使用模式</string>
       </property>
       <widget class="QRadioButton" name="rbSetMatrix">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>设置矩阵输入</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rbSetClick">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>30</y>
          <width>95</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>设置鼠标输入</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
